specification/documentation/APIDescription.md

```diff
@@ -176,6 +176,16 @@ Every request includes an `idempotencyKey` field located in the meta collection
 
 If a request fails, our retry policy will continue to attempt to deliver the callback for a period of 15 minutes.
 
+## Message character limits
+Different character limits apply to each of the communication channels as listed below. NHS Notify will validate that any personalisation fields submitted in the send message request do not exceed these limits but it is the client's responsibility to ensure that when personalisation is combined with any templated text, the channel character limit is not exceeded.
+
+| Channel            | Character Limit |
+|--------------------|-----------------|
+| Email              | 100,000         |
+| Letter             | 1,000           |
```

Comment:
Is letter not 15,000?

Reply:
You are indeed correct. Thanks for spotting! Updated in latest commit